docs/cmf_client/cmf_client.md

```diff
@@ -34,15 +34,15 @@ Required Arguments
   --endpoint-url [endpoint_url]         Specify the endpoint url of minio UI.
   --access-key-id [access_key_id]       Specify Access Key Id.
   --secret-key [secret_key]             Specify Secret Key.
+  --cmf-server-url [cmf_server_url]     Specify cmf-server url. (default: http://127.0.0.1:80)
```

Comment:
why are we documenting the server as required argument ? It should stay optional and take a default value

Reply:
Resolved